msg/Strings.msg

```diff
@@ -0,0 +1,5 @@
+string string_value
+string string_value_default "Foo"
```

Comment:
Please use multiple fields with different default values to cover various cases of escaping: see https://github.com/ros2/rosidl_python/blob/191880315e01f266611dac04da3817b3ef672e81/rosidl_generator_py/msg/Strings.msg

Reply:
Done 6da1125